md/documents.md

```diff
@@ -46,7 +47,18 @@ import org.bonitasoft.engine.bpm.document.DocumentValue
 ]  
 ```
 
-To update a document list, you need to specify the whole list again using a script,. You cannot just add or replace one component of the list.
+To update a document list, you need to write a script in the operation and to use the id of the documents. The script should return a new document list.  
```

Comment:
Is there document id in 7.5?

Reply:
You can use the id manually in 7.5, we do it for you in 7.9
Use the id is the common way to use  documents in 7.x, so we document it.